shared/actions/teams.js

```diff
@@ -438,7 +435,7 @@ const _createNewTeamFromConversation = function*(
 
   if (participants) {
     yield Saga.put(TeamsGen.createSetTeamCreationError({error: ''}))
-    yield Saga.put(TeamsGen.createSetTeamCreationPending({pending: true}))
+    yield Saga.put(WaitingGen.createIncrementWaiting({key: Constants.teamCreationWaitingKey}))
```

Comment:
why not here also

Reply:
did you actually try it with the same key to those other calls, we can easily make a grace period for the spinners to not stop immediately

Reply:
@chrisnojima I didn't notice a visual glitch, though the render function does get called with waiting=false inbetween teamCreate and teamAdd after your change.  (Maybe the return from render() only makes it to a frame on the screen on slower hardware?)

Adding a grace period doesn't feel great to me -- feels like we're just trying to emulate what the code is already doing, but in a way that doesn't have any visibility into what the user's actually waiting on, we could chat more about what we think is the right granularity to spin on.

Reply:
i think if it spins, then stops for a couple of frames (doubtful) and spins some more thats fine. we have plenty of worse situations in the app (no spinners at all) so i think this is a case we can focus on way down the line

Reply:
@chrisnojima Okay, done.